### DISCUSSION TITLE
Add ability to hide comments

### DISCUSSION THREAD

**Discussion by VoltrexKeyva** [community/community#48665] [Mobile]
Hiding comments in issues and pull requests is a useful feature which can be done from the GitHub web interface, but it would be nice to also be able to hide comments in GitHub Mobile as well.

**Comment by HenSquared** [comment]
Hi @VoltrexKeyva, thanks for the feedback! It helps us make the apps better.

I agree, hiding comments is useful! I will open a feature request issue for this and discuss with the team.
